lib/tire/results/collection.rb

```diff
@@ -132,7 +132,7 @@ def __get_results_with_load(hits)
         @response['hits']['hits'].group_by { |item| item['_type'] }.each do |type, items|
           raise NoMethodError, "You have tried to eager load the model instances, " +
                                "but Tire cannot find the model class because " +
-                               "document has no _type property." unless type
+                               "document has no _type property." unless type && ! type.empty?
```

Comment:
Bugfix for `Tire`. The `type` value is initialized to `''` elsewhere, so this error would never be raised. This is fixed mainly so the test will pass. Since it is basically checking that the Elastic response is correct (which it should be), there is little risk here with fixing this bug.